vhpc_toolkit/cluster.py

```diff
@@ -537,44 +539,50 @@ def _add_item_vm(self, cfg, key, value):
             "domain",
             "guest_hostname",
             "vgpu",
-            "sriov_port_group",
             "pvrdma_port_group",
-            "pf",
             "power",
             "dvs_name",
+            "svs_name",
         ]
         float_keys = ["memory"]
-        int_keys = ["cpu", "mmio_size", "cpu_shares", "memory_shares"]
+        int_keys = [
+            "cpu",
+            "mmio_size",
+            "cpu_shares",
+            "memory_shares",
+            "cores_per_socket",
+        ]
         bool_keys = [
             "cpu_reservation",
             "memory_reservation",
             "is_dhcp",
             "linked",
             "instant",
+            "secure_boot",
+            "allow_guest_mtu_change",
         ]
         list_keys = ["device", "dns"]
-        append_keys = ["script"]
-        barrier_keys = ["barrier"]
-        barrier_target = "script"
+        append_keys = ["script", "pf", "sriov_port_group", "sriov_dvs_name"]
```

Comment:
sriov can also attach svs

Reply:
Also make it one key per line.

Reply:
Add `sriov_svs_name`. 
Why these four keys are `append_keys`? Are they optional or necessary?